index.js

```diff
@@ -101,6 +144,13 @@ async function parseSocketConfig (fileContent) {
     throw new ErrorWithCause('Error when parsing socket.yml config', { cause: err })
   }
 
+  if (!parsedContent?.version) {
+    const parsedV1 = await parseV1SocketConfig(parsedContent)
+    if (parsedV1) {
+      return parsedV1
+  }
+}
```

Comment:
```suggestion
    }
  }
```

The indentation needs some fixing here